security/pfSense-pkg-openvpn-client-export/files/usr/local/www/vpn_openvpn_export.php

```diff
@@ -382,7 +381,12 @@
 		exit;
 	}
 }
+?>
 
+<script src="https://code.jquery.com/jquery-1.9.1.js"></script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/css/bootstrap-select.min.css">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/js/bootstrap-select.min.js"></script>
```

Comment:
These must reference the local copies of jquery and bootstrap. The local references are already added by the existing code, so really these three lines may be omitted entirely.